ceramic-config/src/daemon.rs

```diff
@@ -18,6 +18,7 @@ pub struct DaemonIpfsConfig {
     pub host: Option<String>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub pinning_endpoints: Option<Vec<String>>,
+    pub disable_peer_data_sync: Option<bool>,
```

Comment:
If we want to expose this to users will need to add it to https://github.com/ceramicstudio/wheel/blob/main/ceramic-config/src/lib.rs#L13 and expose it with prompts. We can do that in another PR if we want.